app/templates/create.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 rounded bg-white border border-primary" id="createForm" data-link="{{ url_for('create') }}">
```

Comment:
just as a heads up - most likely for the next milestone we will want to use ajax for this, but it works for the MVP and is much simpler

Reply:
if its a seperate page ajax wont be necessary, wtforms does everything needed and displays rerrors on re render of a page.